tests/pr_test.go

```diff
@@ -156,3 +162,76 @@ func TestFSCloudInSchematic(t *testing.T) {
 	err := options.RunSchematicTest()
 	assert.Nil(t, err, "This should not have errored")
 }
+
```

Comment:
We need an upgrade test for every DA variation too. You should be able to just replace the current upgrade test using the example with a DA upgrade test - we don't need both